src/renderer/components/inputs/previews/ImagePreview.tsx

```diff
@@ -83,6 +83,11 @@ const ImagePreview = memo(({ path, schemaId, id }: ImagePreviewProps) => {
                                     ? 'Image preview failed to load, probably unsupported file type.'
                                     : 'File does not exist on the system. Please select a different file.'
                             }
+                            backgroundImage={
+                                img?.channels === 4
```

Comment:
I don't think that's necessary. If the image isn't transparent, nobody will see the BG image. It's also more efficient without the conditional, since the CSS-in-JS library doesn't have to update the styles.

Reply:
You actually kinda can see it around the border... It's weird. Change it locally to do it all the time and you'll notice the difference

Reply:
Good point.